cdc/processor/pipeline/sink.go

```diff
@@ -194,6 +192,7 @@ func (n *sinkNode) addRowToBuffer(ctx context.Context, event *model.PolymorphicE
 		return nil
 	}
 
+	rows := make([]*model.RowChangedEvent, 0, 2)
```

Comment:
Whether this slice will be allocated to heap and trigger frequent gc ?

Reply:
Why might frequent gc be triggered here?

Reply:
It is allocated to heap. Could you add a benchmark and show the performance impact? I think 10% regression may be acceptable given that the bottleneck is actually in sink IO not CPU.

Reply:
Each round runs for 30s:
slice: BenchmarkEmitRowToSink-16 1202 74805793 ns/op
direct send: BenchmarkEmitRowToSink-16 914 113538113 ns/op

Reply:
I will remove it.